runtime/queries/php/highlights.scm

```diff
@@ -8,6 +8,14 @@
 (named_type (name) @type) @type
 (named_type (qualified_name) @type) @type
 
+(namespace_definition
+  name: (namespace_name(name)) @namespace
+)
+
+(subscript_expression
+  (variable_name(name) @constant)
```

Comment:
Is this really a constant?

Reply:
it's not constant, it is name Superglobals in php, like constant ,is global . after php 8.1 ,  some become read-only like $GLOBALS.  Superglobals is diffent with variable , In practical applications, most of them are read-only, except magic.  It can be regarded as special constant。

may be mark this with `variable.builtin`

Reply:
PHP extensions will also append some Superglobals.
I mark `constant.builtin`.